package.json

```diff
@@ -84,20 +86,27 @@
     "acorn": "^7.4.0",
     "benchmark-suite": "^0.1.8",
     "commander": "^9.4.1",
+    "commitizen": "^4.2.6",
+    "commitlint": "^17.3.0",
     "eslint": "7.2.0",
     "eslint-config-airbnb": "^18.2.0",
     "eslint-config-prettier": "^6.10.1",
     "eslint-plugin-import": "^2.21.2",
     "eslint-plugin-json-format": "^2.0.1",
     "eslint-plugin-prettier": "^3.1.4",
+    "husky": "^8.0.2",
     "jest": "^26.4.0",
+    "jest-date-mock": "^1.0.8",
     "jest-when": "^2.7.2",
+    "lint-staged": "^13.1.0",
+    "madge": "^5.0.1",
     "mocked-env": "^1.3.2",
     "pre-commit": "^1.2.2",
-    "prettier": "^1.18.2"
+    "prettier": "^1.18.2",
+    "standard-version": "^9.5.0"
```

Comment:
https://www.npmjs.com/package/standard-version
It says that `standard-version` is deprecated. Should we take this out ?

Reply:
True, but its current version exactly fits our use case. We can check for alternatives during implementation.